docs/eln/ui/Inventory/create_inventroy_entry.mdx

```diff
@@ -0,0 +1,82 @@
+* The <Btn mixed={["Choose Language of SDS"]} color={"secondary"} /> dropdown list offers three language options: English, German, and French. If no results are found with the chosen language, results will be displayed with the default query language (English).
+* After selecting the search parameters, users can initiate the search query by clicking the <Btn mixed={["Search for SDS"]} color={"secondary"} /> button.
+* The search results will be displayed, and users can click on the results *tag link* to open a new tab directing them to the vendor's safety data sheet.
+* It is recommended for users or administrators to first verify if the displayed results are associated with the desired chemical.
+* If results match the specific chemical, users can save the safety sheet to the Electronic Laboratory Notebook (ELN) by clicking the save button next to the relevant result tag. A green check mark icon will appear next to result element tag, if SDS is saved successfully. 
+* Once saved, users can initiate additional search queries to retrieve safety phrases or search for chemical and physical properties related to the chemical.
+* Users have the option to delete the saved data sheet by clicking the delete icon <Btn mixed={[faTrash]} color={"danger"}/> button.
+* The <Btn mixed={["Search for SDS"]} color={"secondary"} /> button will be disabled, once one SDS is saved. To enable another search query, user have to delete any saved SDS first.
+* Additionally, users can access the vendor's product webpage by clicking the external link icon <Btn mixed={[faExternalLink]} color={"secondary"} /> button.
+* User can query the safety phrases for this chemical from vendor by clicking the <Btn mixed={["fetch safety phrases"]} color={"secondary"} /> button. Results render below as depicted in the image below.
+
+![safety bar](/img/inventoryTab_safety_list.png)
+
+* The <Btn mixed={["fetch Chemical Properties"]} color={"secondary"} /> button enable users to make a search query for chemical and physical properties related to the chemical and save it to respective fields in sample properties tab. 
+Sometimes fetched results can not be saved automatically if fetched values can not be accepted by ELN for some fields. User in this case can view and modify entered values manually.
+Fetched properties can be viewed by clicking the results icon <Btn mixed={[faFileText]} color={"secondary"} /> button next to fetch button. An example of fetch chemical properties process is shown below.
+
+![Safety List](/img/fetch_chemical_properties_chemical.gif)
+
+## Location and Information List:
+The Location field is devided into 2 subsections `host location, host group ` which reflects information about owner of chemical and `current location, current group` in case of chemical is borrowed by another group/person. Both subsections have overlapping fields.
```

Comment:
"host location, host group"  -- extra space before ending annotation

"in case if* chemical" sounds better?